paper-drawer-panel.html

```diff
@@ -230,6 +230,8 @@
       }
 
       .left-drawer.narrow-layout > #drawer:not(.iron-selected) {
+        visibility: hidden;
+
```

Comment:
This will make swipe from the edge not working.  We need to add the same we have for the right-drawer: https://github.com/PolymerElements/paper-drawer-panel/blob/a4980687329d36cfbda3fba9b0a7e888a239690b/paper-drawer-panel.html#L247